apps/settings/elements/browsing_privacy.html

```diff
@@ -21,6 +21,14 @@ <h1 data-l10n-id="browsingPrivacy-header"></h1>
             <details data-l10n-id="privateBrowsingDefaultDetails"></details>
           </gaia-switch>
         </li>
+        <li>
+          <gaia-switch name="privacy.trackingprotection.enabled">
+            <label data-l10n-id="trackingProtection"></label>
+            <details data-l10n-id="trackingProtectionDetails"></details>
+            <a data-l10n-id="learn-more"
```

Comment:
It's might be possible, but it requires much less gaia_switch restructuring to do it this way. I started with the link in details, but realized it was easier to put it outside. For instance, if you put the link inside, you would need a text wrapping element for the details (for localization), or you could put the markup in the localization string itself. Making gaia_switch work both with this wrapper element and without is kinda painful.

The other advantage of doing it this way is if you don't include the link, the contect select="a" inserts an empty string into the DOM, and all the old components without the link can keep the same structure as before.